docs/rule/require-input-label.md

```diff
@@ -81,6 +81,14 @@ This rule **allows** the following:
 * another option is to add an aria-label to the input element.
 * wrapping the input element in a label element is also allowed; however this is less flexible for styling purposes, so use with awareness.
 
+## Configuration
+
+ The following values are valid configuration:
+
+* boolean - `true` to enable / `false` to disable
+* object -- An object with the following keys:
+  * `allow` -- An array of component names for that may be used as label replacement
```

Comment:
Can you rename `allow` to `labelComponentNames` for greater clarity?

Reply:
Or maybe `customLabelComponents`?

Reply:
I think this is too long names, you have to read doc and copy it, quite impossible to remember

Reply:
My concerns are:

* It's not clear what is being allowed by this `allow` option. Someone reading this option will have to look at the doc to understand what it refers to. I think option names should be able to stand on their own ("self-documenting").
* What if there are other types of things we want to allow in the future besides just component names? Using `allow` will make it harder for us to add other types of "allowed" options in the future.

I don't think length matters. People will not be typing this option name often.

Reply:
How about “allowTags”?

Reply:
Okay. As long as you think that it will be clear that this refers to custom label component tags/names.

Reply:
@bmish looks like `labelTags` fits here :)